index.md

```diff
@@ -6,16 +6,43 @@ permalink: /events/
 active_sub_menu_id: shared_mnu_events_events
 ---
 
-## Upcoming Conferences & Events
-Meet our team and join our sessions at the following conferences and meetups.
+## Recent Events
+
+### Confoo Vancouver : December 7th 2016
+
+- [Caching in applications still matters !](https://confoo.ca/en/yvr2016/session/caching-in-applications-still-matters), by [Anthony Dahanne](https://confoo.ca/en/speaker/anthony-dahanne)
+
+### Victoria JUG : December 6th 2016
+
+- [Caching in applications still matters !](https://www.eventbrite.ca/e/anthony-dahanne-xmatters-registration-29901608516), by [Anthony Dahanne](http://blog.dahanne.net)
+
+### Montreal JUG : November 24th 2016
+
+- [Pourquoi le caching est toujours si important ?](http://www.montreal-jug.org/meetup/caching/), by [Anthony Dahanne](http://blog.dahanne.net)
+
+### Devoxx Belgium : November 7-11 2016 : Antwerp, Belgium
+
+The Terracotta / Software AG team will be out in full force.  Join us at our booth, see demos, get technical assistance, and win prizes.
+
+Also, don't forget to catch talks given by the Ehcache / Terracotta / Software AG crew!
+
+#### Ehcache and Terracotta Server talks at Devoxx Belgium 2016
+
+- [Ehcache out of his element](http://cfp.devoxx.be/2016/talk/AUX-3854/Ehcache_out_of_his_Element), by [Louis Jacomet](http://cfp.devoxx.be/2016/speaker/louis_jacomet)
+- [Terracotta Ehcache: Simpler, faster, distributed](http://cfp.devoxx.be/2016/talk/EMH-3283/Terracotta_Ehcache:_simpler,_faster,_distributed), by [Anthony Dahanne](http://cfp.devoxx.be/2016/speaker/anthony_dahanne)
+
+### Devoxx Morocco : November 1st 2016 : Casablanca, Morocco
```

Comment:
Content is incorrect - get rid of the section for now, will do a separate PR later